sockeye/model.py

```diff
@@ -56,8 +56,8 @@ def __init__(self,
                  max_seq_len_target: int,
                  vocab_source_size: int,
                  vocab_target_size: int,
-                 config_encoder: encoder.EncoderConfig,
-                 config_decoder: decoder.RecurrentDecoderConfig,
+                 config_encoder: Config,
```

Comment:
EncoderConfig was too narrowly typed so I removed it.
RecurrentDecoderConfig was just wrong as it might also be a TransformerConfig (which is the same for both encoder and decoder).
So the next common type is Config which is fine I think.